src/grasp_generator.cpp

```diff
@@ -798,21 +802,23 @@ double GraspGenerator::scoreGrasp(const Eigen::Affine3d& grasp_pose, const Grasp
 double GraspGenerator::scoreSuctionGrasp(const Eigen::Affine3d& grasp_pose, const GraspDataPtr& grasp_data)
 {
   ROS_DEBUG_STREAM_NAMED("grasp_generator.scoreGrasp",
-                         "Scoring grasp at: ("
-                             << grasp_pose.translation().x() << ", " << grasp_pose.translation().y() << ", "
-                             << grasp_pose.translation().z() << ") (" << grasp_pose.rotation().eulerAngles(0, 1, 2)(0)
-                             << "," << grasp_pose.rotation().eulerAngles(0, 1, 2)(1) << ","
-                             << grasp_pose.rotation().eulerAngles(0, 1, 2)(2) << ") vs ("
-                             << ideal_grasp_pose_.translation().x() << ", " << ideal_grasp_pose_.translation().y()
-                             << ", " << ideal_grasp_pose_.translation().z() << ") ("
-                             << ideal_grasp_pose_.rotation().eulerAngles(0, 1, 2)(0) << ","
-                             << ideal_grasp_pose_.rotation().eulerAngles(0, 1, 2)(1) << ","
+                         "Scoring grasp at: \n\tpose:  ("
+                             << grasp_pose.translation().x() << ",\t" << grasp_pose.translation().y() << ",\t"
+                             << grasp_pose.translation().z() << ")\t(" << grasp_pose.rotation().eulerAngles(0, 1, 2)(0)
+                             << ",\t" << grasp_pose.rotation().eulerAngles(0, 1, 2)(1) << ",\t"
+                             << grasp_pose.rotation().eulerAngles(0, 1, 2)(2) << ")\n\tideal: ("
+                             << ideal_grasp_pose_.translation().x() << ",\t" << ideal_grasp_pose_.translation().y()
+                             << ",\t" << ideal_grasp_pose_.translation().z() << ")\t("
+                             << ideal_grasp_pose_.rotation().eulerAngles(0, 1, 2)(0) << ",\t"
+                             << ideal_grasp_pose_.rotation().eulerAngles(0, 1, 2)(1) << ",\t"
                              << ideal_grasp_pose_.rotation().eulerAngles(0, 1, 2)(2) << ")");
 
   // get portion of score based on the orientation
   Eigen::Vector3d orientation_scores = GraspScorer::scoreRotationsFromDesired(grasp_pose, ideal_grasp_pose_);
 
   // get portion of score based on the translation
+  // Eigen::Vector3d translation_scores =
```

Comment:
Will this be added later?

Reply:
No, this is the finger grasp scoring method. I deleted the dead code